fe/fe-core/src/main/java/org/apache/doris/mysql/MysqlSerializer.java

```diff
@@ -298,6 +298,8 @@ public int getMysqlDecimals(Type type) {
             case DECIMAL64:
             case DECIMAL128:
             case DECIMAL256:
+            case TIMEV2:
+            case DATETIMEV2:
```

Comment:
should not be in `getMysqlDecimals` for TIMEV2 and DATETIMEV2, function used for decimal

Reply:
mysql use this for datatime
![image](https://github.com/apache/doris/assets/22926391/9cf81389-7e99-474b-a25d-a908864ec379)
<img width="1066" alt="image" src="https://github.com/apache/doris/assets/22926391/f7d7fa31-f6a6-4455-8305-25d4cf46f783">